src/kernels/composable_kernel/include/utility/tuple_ext.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef CK_TUPLE_EXT_HPP_
+#define CK_TUPLE_EXT_HPP_
```

Comment:
Please rename this file to ```multi_index_transform_helper.hpp``` and move it under ```composable_kernel/include/tensor_descriptor/```

Reply:
Agree !

Reply:
Check https://github.com/ROCmSoftwarePlatform/MIOpen/pull/322/commits/08ca8ac8606b4f672065a4918044506c3f15e6e1